add-on/src/options/options.html

```diff
@@ -1,56 +1,167 @@
 <!DOCTYPE html>
-
 <html>
   <head>
     <meta charset="utf-8">
     <style type="text/css">
-      input { display: inline-block; } /* fix for https://github.com/ipfs/ipfs-companion/issues/257 */
+      fieldset {
+        border: none;
+        border-top: 3px solid #3E9398;
+        display: flex;
+        flex-direction: column;
+        margin-bottom: .5em;
+        min-width: 40em;
+      }
+      legend {
+        letter-spacing: .15em;
+        text-transform: uppercase;
+        padding: .5em;
+        font-weight: bold;
+      }
+      fieldset > div {
+        padding: 1em;
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+      }
+      fieldset > div + div {
+        border-top: 1px solid #ccc;
+      }
+      fieldset > div > * {
+        align-self: center;
+
+      }
+      label {
+        flex: 1;
+      }
+      label > dl > dd {
+        opacity: .5;
+        font-size: .9em;
+        margin-left: 2em;
+        margin-right: .2em;
+      }
+      div:hover {
+        background-color: rgba(62, 148, 152, 0.05);
+      }
+      div:hover > label > dl > dd {
+        opacity: 1;
+      }
+      input, textarea {
+        flex: 1;
+        padding: .5em;
+        font-family: monospace;
+      }
+      input[type=text],
+      input[type=url],
+      input[type=number],
+      textarea {
+        -webkit-transition: all 0.30s ease-in-out;
+        -moz-transition: all 0.30s ease-in-out;
+        border: 1px solid #ccc;
+      }
+      input[type=text]:focus,
+      input[type=url]:focus,
+      input[type=number]:focus,
+      textarea:focus {
+        outline: none;
+        border: 1px solid #3E9398;
+        box-shadow: 0 0 5px #5FCBCF;
+      }
     </style>
+    <script src="../lib/npm/browser-polyfill.min.js"></script>
   </head>
-
   <body>
-
     <form>
-      <fieldset style="line-height: 2em; font-family: sans-serif;">
-        <legend>Basic</legend>
-
-
-        <p><label>Public Gateways Hosts <input type="text" id="publicGateways" size="40" required placeholder="List of hostnames separated with spaces"/></label></p>
-
-        <p><label>Custom Gateway URL <input type="url" id="customGatewayUrl" size="40" inputmode="url" required pattern="https?://.+"/></label></p>
-
-
-        <p><label>IPFS API URL <input type="url" id="ipfsApiUrl" size="40" inputmode="url" required pattern="https?://.+"/></label></p>
-
-        <p><label>Redirect IPFS Requests to the Custom Gateway  <input type="checkbox" id="useCustomGateway" /></label></p>
-
-        <p><label>Automatic Mode <small>(Redirect only when IPFS API is Online and node has peers)</small> <input type="checkbox" id="automaticMode" /></label></p>
-
+      <fieldset>
+        <legend>Gateways</legend>
+        <div>
+          <label for="publicGateways">
+            <dl>
+              <dt>Public Gateways</dt>
+              <dd>list of hostnames known for exposing IPFS resources under <code>/ipfs/</code> path</dd>
```

Comment:
Recommend rewording:
`<dd>List of hostnames known to expose IPFS under <code>/ipfs/</code></dd>`

Reply:
🍰